src/azure-cli/HISTORY.rst

```diff
@@ -7,6 +7,10 @@ Release History
 
 * Add support for importing/exporting feature flags
 
+**AppService**
+
+* Fix issue #7154: webapp:Single quotes instead of back ticks
```

Comment:
can you say updating documentation for command <> to use quoted instead of back ticks?

Reply:
It's the other way around but yes :)